src/commands/lambda/flare.ts

```diff
@@ -771,6 +784,75 @@ export const sleep = async (ms: number) => {
   await new Promise((resolve) => setTimeout(resolve, ms))
 }
 
+/**
+ * Get the framework used based on the files in the directory
+ * @returns the framework used or undefined if no framework is found
+ */
+export const getFramework = () => {
+  const files = new Set(fs.readdirSync(process.cwd()))
+
+  // Check for serverless framework
+  const serverlessFiles = ['serverless.yml', 'serverless.yaml']
+  for (const file of serverlessFiles) {
+    if (files.has(file)) {
+      return 'Serverless Framework'
+    }
+  }
+
+  // Check for AWS CDK
+  if (files.has('cdk.json')) {
+    return 'AWS CDK'
+  }
+
+  // Check for AWS CloudFormation
+  const cloudFormationFiles = ['template.yml', 'template.yaml']
+  for (const file of cloudFormationFiles) {
+    if (files.has(file)) {
+      return 'AWS CloudFormation'
+    }
+  }
+
+  return 'Unknown'
+}
+
+/**
+ * Generate the insights file
+ * @param insightsFilePath path to the insights file
+ * @param isDryRun whether or not this is a dry run
+ * @param config Lambda function configuration
+ */
+export const generateInsightsFile = (insightsFilePath: string, isDryRun: boolean, config: FunctionConfiguration) => {
+  const lines: string[] = []
+  lines.push('_Autogenerated file from `lambda flare`_')
+  if (isDryRun) {
+    lines.push('</br>_This command was run in dry mode._')
```

Comment:
What's the difference between `</br>` and `\n`?

Reply:
Good question! In markdown, `</br>` creates a small line break. Using `\n` creates a bigger gap between the two lines.

With `</br>`:
<img width="287" alt="Screenshot 2023-07-25 at 2 55 11 PM" src="https://github.com/DataDog/datadog-ci/assets/29668820/942ada1a-fc74-49b8-8725-03ca9a5448e8">

With new lines:
<img width="298" alt="Screenshot 2023-07-25 at 2 55 29 PM" src="https://github.com/DataDog/datadog-ci/assets/29668820/d3a75cf2-b342-4189-8ac7-b970d1f7607c">